hles/orchestration/templates/ingest-hles-baseline.yaml

```
@@ -0,0 +1,101 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: ingest-hles-baseline
+spec:
+  templates:
+    - name: main
+      inputs:
+        parameters:
+          - name: gcs-prefix
+          {{- $gcsPrefix := "{{inputs.parameters.gcs-prefix}}" }}
+      dag:
+        tasks:
+          - name: run-extraction
+            template: run-extraction-pipeline
+            arguments:
+              parameters:
+                - name: output-prefix
+                  value: '{{ printf "%s/raw" $gcsPrefix }}'
+          - name: run-transformation
+            dependencies: [run-extraction]
+            template: run-transformation-pipeline
+            arguments:
+              parameters:
+                - name: input-prefix
+                  value: '{{ printf "%s/raw" $gcsPrefix }}'
+                - name: output-prefix
+                  value: '{{ printf "%s/processed" $gcsPrefix }}'
+
+
+    {{- /* Version used for both Dataflow docker images. */}}
+    {{- $version := default "latest" .Chart.AppVersion }}
+    {{- $bucket := .Values.gcs.bucketName }}
+    ##
+    ## Template used to launch a Dataflow processing job that pulls
+    ## all relevant metadata from REDCap.
+    ##
+    - name: run-extraction-pipeline
+      inputs:
+        parameters:
+          - name: output-prefix
+          {{- $outputPrefix := "{{inputs.parameters.output-prefix}}" }}
+      container:
+        image: us.gcr.io/broad-dsp-gcr-public/dog-aging-extraction-pipeline:{{ $version }}
```

```suggestion
        image: us.gcr.io/broad-dsp-gcr-public/dog-aging-extraction:{{ $version }}
```
I think, based on the sbt project names